Dockerfile

```diff
@@ -44,29 +45,28 @@ RUN apt install -y libxkbcommon0 \
     xvfb
 
 COPY ./deploy/requirements/* /opt/
-RUN curl -sS https://bootstrap.pypa.io/get-pip.py | python3.10
-RUN pip3 install -r /opt/click.txt -r /opt/prod.txt
-COPY ./deploy/oz/run-full-test-suite.sh /opt/neon-tests/
+
+RUN curl -sS https://bootstrap.pypa.io/get-pip.py | python3.10 && \
+    pip3 install uv && \
+    uv venv
+
+ENV VIRTUAL_ENV=/.venv
```

Comment:
why we need this?

Reply:
To make the virtual environment visible for all the docker steps during the build